src/main/java/com/ncedu/cheetahtest/service/user/UserServiceImpl.java

```diff
@@ -60,4 +60,39 @@ public void makeTokenExpired(ResetToken resetToken) {
         resetTokenDao.makeTokenExpired(resetToken);
     }
 
+    @Override
+    @Transactional
+    public boolean editUser(User user) {
+        if(user.getRole().equals("admin")) {
+            userDao.editUser(user);
+            return true;
+        }
```

Comment:
Security checks should be performed by Spring Security. For example, by using @PreAuthorize